### DISCUSSION TITLE
Routine Load  With Local: Bad Message s

### DISCUSSION THREAD

**Discussion by SunnyPirate** [apache/doris#36475] [RFC]
Doris : 2.0.11
Kafka : 2.1.1-1.1.1 deployed on k8s and export with nodeportsd
I start routine load On doris and get message below
org.apache.doris.common.LoadException: errCode = 2, detailMessage = Failed to get latest offsets of kafka topic: action_9. error: errCode = 2, detailMessage = failed to get latest offsets: [(172.21.80.142)[INTERNAL_ERROR]failed to get latest offset for partition: 1, err: Local: Bad message format]

**Comment by sollhui** [comment]
Did you use a domain name to link Kafka? If so, did you configure domain name mapping in etc/hosts ?

**Comment by ixzc** [comment]
you can refer this: https://github.com/apache/doris/discussions/15536
